### ISSUE TITLE
Don't close on a busted release.

### ISSUE THREAD

**Issue by atz** [projecthydra/jettywrapper#58]
Deprecation or obsolescence is one thing, but leaving a gem in a state where the most recently published version is uniformly broken is another.  `2.0.3` is not useful for any purpose, since it fails to install its own dependencies and dies as soon as called (see #39).  This appears to have been fixed, but is not released!

Can we at least get CI passing and leave a valid most-recent release?  Then I can officially stop caring about jettywrapper.

**Comment by cbeer** [comment]
Sounds like a great idea. I'd guess most of the hydra community has moved beyond jettywrapper, so I'd strongly encourage you to have at it.

**Comment by atz** [comment]
Most of our infrastructure has not.

**Comment by atz** [comment]
Perhaps we should yank `2.0.3` as well?

**Comment by atz** [comment]
Note: when I re-run previously passing CI builds, they now fail.

**Comment by atz** [comment]
OK, fixed that with #59.  Now it is just jettywrapper's own problems.

**Comment by atz** [comment]
Released `2.0.4`

**Comment by hackmastera** [comment]
@atz thank you for this. sufia 6.x uses jettywrapper by default and migration tools are not yet complete. @cbeer I do not think the community has moved on. Even as someone actively contributing to core I needed to spin up a clean sufia 6 instance yesterday to get some export data for my migration work.

**Comment by jcoyne** [comment]
@hackmastera I'm don't think Sufia 6 has any dependency on jettywrapper. It only has a dependency on a running version of solr and fcrepo. It should work just the same with solr_wrapper/fcrepo_wrapper.

**Comment by hackmastera** [comment]
@jcoyne sure. i didn't say it's a dependency. It's a default, which currently installs on a fresh build and probably is in use in a bunch of existing builds.

But I'm not trying to get hung up on jettywrapper per se. just assert the need for perspective in determining what is/is not relevant or in-use in the community.

**Comment by jcoyne** [comment]
@hackmastera I'm saying it probably shouldn't be the default.  Do you know where is the default coming from?

**Comment by hackmastera** [comment]
@jcoyne here's what i can tell you. I needed an export from a clean sufia 6.7 instance to use in testing import. I cloned sufia and checked out 6.7. I tried to build an internal_test_app but engine_cart hung indefinitely. So I created a new rails project and installed sufia 6.7 manually, ran it according to the readme at that version. Created my test object and exported it and wrote my test, hooray!
